src/evaluator/from-flake.nix

```diff
@@ -0,0 +1,22 @@
+{ inputs
+, system
+, ...
+}:
+let
+  evaluatorOutputs = import ./default.nix {
+    # TODO: How to compute this on Flakes?
+    makesExecutionId = "123";
+    makesSrc = inputs.makes.outPath;
+    projectSrc = inputs.self.sourceInfo.outPath;
+    # TODO: Make it mutable
+    projectSrcMutable = inputs.self.sourceInfo.outPath;
```

Comment:
Same here:

```nix
{
  description = "";
  inputs = {};
  outputs = {
    dyn.makesExecutionId = ""; # injected values by cli wrapper
    dyn.projectSrcMutable = ""; # injected values by cli wrapper
  };
}
```

Reply:
But that might make `self.rev` dirty, which is another issue...

Reply:
This is a little more of a headache, since it is really useful for a task runner to actually run tasks on the repo, such as code generation...

Reply:
it may be simple to solve, I'm gonna leave this to the end as it is not a core feature,

I'll be iterating next on how to inject apps into the framework,
then I'll move to https://github.com/fluidattacks/makes/issues/545
and finally the details (cli, mutability, etc)